examples/js/app.tsx

```diff
@@ -30,11 +30,13 @@ insightsClient('init', { appId, apiKey });
 
 const algoliaInsightsPlugin = createAlgoliaInsightsPlugin({ insightsClient });
 const recentSearchesPlugin = createLocalStorageRecentSearchesPlugin({
+  sourceId: 'recentSearchesPlugin',
```

Comment:
We'll provide a `transformSource` param instead of the current `getTemplates` param which will allow to edit the `sourceId` of the source, so we shouldn't add a `sourceId` param here, but rather default it in the plugin itself.